data/nlu/faq.md

```diff
@@ -0,0 +1,266 @@
+## intent:ask_faq_channels
+- No, I mean how it is possible to use Skype as channel?
+- what channels do you support?
+- what chat channels does rasa uses
+- Channels
+- channels supported by Rasa
+- what are the channels Rasa NLU supports
+- which UI channel is used by Rasa ?
+- which messaging channels does rasa support?
+- which particular messaging channels are supported by rasa?
+- what are the messaging channels supported by rasa?
+- which messaging channels are supported by rasa?
+- rasa supports which messaging channels?
+- which messaging channels can I use with rasa?
+- which messaging channels can be used with rasa?
+- which messaging channels are compatible with rasa?
+- what are the messaging channels that can be used with rasa?
+- rasa can use which different messaging channels?
+- WHAT MESSAGING PORTALS DOES RASA SUPPORT?
+- DOES RASA SUPPORT MESSENGER?
+- DOES RASA SUPPORT THE WHATS APP?
+- DOES RASA SUPPORT SMS?
+- Do you know which messaging channels rasa supports?
+- Can you tell which messaging channels does rasa support?
+- Rasa supports some messaging channels, what are those?
+- What channels of communication does rasa support?
+
+## intent:ask_faq_community_size
+- How large is the Rasa community
+- how big is your community?
+- how many forum members do you have
+- do you have a large community
+- how many people are using Rasa
+
+## intent:ask_faq_differencecorenlu
```

Comment:
do we have anything for the difference between rasa and rasa x?

Reply:
I didn't include it because I felt it was more likely to want to compare Core vs NLU than Rasa vs Rasa X, but I guess you're right.